sonar-core/src/main/java/org/sonar/core/issue/DefaultIssue.java

```diff
@@ -341,15 +340,14 @@ public DefaultIssue setResolution(@Nullable String s) {
     return this;
   }
 
+  /**
+   * @deprecated since 5.5, manual issue feature has been dropped.
+   */
+  @Deprecated
   @Override
   @CheckForNull
   public String reporter() {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)